src/config/config.rs

```diff
@@ -387,6 +390,25 @@ pub struct Http {
   pub group_by: Vec<String>,
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug, Default, PartialEq, Eq)]
+pub enum ExprBody {
+    #[default]
+    None,
```

Comment:
Why is `None` required?

Reply:
Removed. I added it to be able to derive `Default`, but that isn't required.